src/apps/frontend/types/account.ts

```diff
@@ -1,15 +1,18 @@
-import { JsonObject } from './common-types';
+import { PhoneNumber } from './auth';
+import { JsonObject, Nullable } from './common-types';
 
 export class Account {
   id: string;
   firstName: string;
   lastName: string;
+  phoneNumber: Nullable<PhoneNumber>;
```

Comment:
Instead can we make this as an optional parameter like `phoneNumber?: PhoneNumber;`.

Reply:
Yeah, I think this is better.